docs/azure/best-practices/azure-services.md

```diff
@@ -32,3 +32,53 @@ When deploying an [Azure Database for PostgreSQL Flexible server](https://learn.
 ![Azure Database for PostgreSQL Flexible Server - Networking Connectivity](../images/azure-db-postgres-flexible-server-network-connectivity.png "Azure Database for PostgreSQL Flexible Server - Networking Connectivity")
 
 Although the connectivity method says "**public access**", this is the option you want to select, in order to be able to [add private endpoint connections](https://learn.microsoft.com/en-us/azure/postgresql/flexible-server/how-to-networking-servers-deployed-public-access-add-private-endpoint?tabs=portal-add-private-endpoint-connections).
+
+## Azure Databricks and Unity Catalog
+
+### Overview
+
+[Azure Databricks](https://learn.microsoft.com/en-us/azure/databricks/introduction/) is **available for use** in our environment, while [Unity Catalog](https://learn.microsoft.com/en-us/azure/databricks/data-governance/unity-catalog/) (a governance layer) is currently not enabled. This document outlines the distinction between these components, clarifies where each resides and is managed, and explains the current status of Unity Catalog.
+
+### Key Distinctions
+
+| Feature                        | Owned/Managed by         | Lives in Azure? | Lives in Databricks?   |
+| ------------------------------ | ------------------------ | --------------- | ---------------------- |
+| **Azure Databricks Workspace** | Azure Resource Manager   | ✅ Yes          | ✅ Yes (shared)        |
+| **Unity Catalog**              | Databricks Control Plane | ❌ No           | ✅ Yes                 |
+| **Metastore**                  | Databricks Account       | ❌ No           | ✅ Yes                 |
+| **Azure AD (Entra ID)**        | Microsoft                | ✅ Yes          | Used via federation    |
+| **Data Storage (e.g., ADLS)**  | Azure                    | ✅ Yes          | Accessed by Databricks |
+
+### How Unity Catalog Works
+
+- **Unity Catalog** is a centralized **data governance layer** for Databricks that enables fine-grained access control across all data assets (tables, views, files).
+- It is **not an Azure-native service**. It runs on the **Databricks control plane**, and is provisioned/managed through [https://accounts.azuredatabricks.net
+  ](https://accounts.azuredatabricks.net).
+- It can enforce **identity-based policies**, using **Microsoft Entra ID** groups via SCIM federation or manual assignment.
+- It governs access to data even when the actual storage resides in **Azure Data Lake Storage Gen2** or other Azure-native services.
+
+### Current Status in Our Environment
+
+!!! info "Azure Databricks is available, but Unity Catalog is currently unavailable for use."
+
+- While technically supported within Azure Databricks, **we do not currently have an assigned owner or governance process** for Unity Catalog in our environment.
```

Comment:
If you want these bullet within the callout box, you need to indent them. Also, remove the blank line after the header/title of the box.